### PR TITLE
Update normalization.py - fixed tf.layer.batch_normalization deprecate note

### PR DESCRIPTION
**Pull request by jtressle**
tf.layer.batch_normalization referenced incorrect keras.layer.BatchNormalization function.

### CODE REVIEW

**Comment by jtressle** [comment]
I signed it!

**Review by mihaimaruseac — APPROVED** [review]

**Comment by mihaimaruseac** [comment]
Thank you for catching this
